src/wallet/wallet2.cpp

```diff
@@ -5037,6 +5044,32 @@ uint64_t wallet2::adjust_mixin(uint64_t mixin)
   return mixin;
 }
 //----------------------------------------------------------------------------------------------------
+uint32_t wallet2::adjust_priority(uint32_t priority)
+{
+  if (priority == 0 && get_default_priority() != 1 && auto_low_priority())
+  {
+    try
+    {
+      const double fee_level = get_fee_multiplier(1) * get_per_kb_fee() / (double)1024;
+      const std::vector<std::pair<uint64_t, uint64_t>> blocks = estimate_backlog({std::make_pair(fee_level, fee_level)});
+      if (blocks.size() != 1)
+      {
+        MERROR("bad estimated backlog array size");
+      }
+      else if (blocks[0].first == 0)
+      {
+        MINFO("There's no backlog for the low priority fee level, so we use the low priority.");
+        priority = 1;
```

Comment:
Wouldn't it be a good idea to use `priority = Priority.Priority_Low` instead?

Reply:
I don't see any merit in doing so. `Priority_Low` is only defined for the wallet API, and the API needs to reference the core code, not the other way around. The mapping between `uint32_t priority` and the actual multiplier is defined in `wallet2::get_fee_multiplier`.